### DISCUSSION TITLE
Spring vulnerability

### DISCUSSION THREAD

**Discussion by Kulakivsky** [ouuan/ouuan#47] [Q&A (English)] [answered]
Hi @ouuan, sorry for bothering you but I have a question regarding this vulnerability: 
https://spring.io/security/cve-2024-38807 
I am trying to find the solution to this problem in the list of bug fixes for 3.2.9 spring-boot release but unfortunately nothing seems to be related. Can you please guide me to commit that fixed this issue?
Thank you for your time in advance.

**Comment by ouuan** [comment, marked as answer]
https://github.com/spring-projects/spring-boot/commit/0b24ee857189e139f48826bf2aef10ae8680c11b
